### ISSUE TITLE
Swing Bone Export Bug

### ISSUE THREAD

**Issue by JArmstrongArt** [ssbucarlos/smash-ultimate-blender#164]
[Download Specimen of Files](https://github.com/ssbucarlos/smash-ultimate-blender/files/9759546/Specimen.zip)
Another issue I was recommended to list, though I'm not entirely sure it was this tool's fault myself.
I used this tool to export a new skeleton for my model in which all I did after importing this skeleton beforehand with this very same tool is added a few swing bones.
I updated my swing.prc to match, and the swing bones work fine.
However, now that I have updated my skeleton, the skeleton stops animating right when moving on the ground as shown in the image below.
In the air, it looks fine, but on the ground, this happens.
I did not alter the skeleton in any way other than these swing bones.
![image](https://user-images.githubusercontent.com/47801313/195190944-24bd0314-3429-4d07-9d13-64a16d5cbb54.png)

**Comment by ssbucarlos** [comment]
The behavior seems to indicate an issue with the .nusktb, and since only swing bones were added then its most likely an issue with the recently re-written bone re-ordering code. This is a high priority issue imo so i will look into it asap. will let u know when its fixed

**Comment by ssbucarlos** [comment]
in the meantime u can also manually re-order bones in ssbh_editor, i recommend moving the swing bones down the list until they are with the vanilla swing bones

**Comment by JArmstrongArt** [comment]
UPDATE: Just out of curiosity, I rolled back my plugin install to commit [27505ff](https://github.com/ssbucarlos/smash-ultimate-blender/tree/27505ff68213673e789447003eb0931a09f3dc7d), before you added [that recent merge with the helper bone stuff](https://github.com/ssbucarlos/smash-ultimate-blender/tree/153f58ebc819cc167738888186a3256cd282c1fa). However, despite a re-export with this rolled-back version, the problem persists after checking my bone order on both my old and new skeleton in SSBH Editor. I only tried this because I know you mentioned to me that a recent push might have caused this.
